lib/active_job/queue_adapters/cloudtasker_adapter/worker.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# ActiveJob docs: http://edgeguides.rubyonrails.org/active_job_basics.html
+# Example adapters ref: https://github.com/rails/rails/tree/master/activejob/lib/active_job/queue_adapters
+
+module ActiveJob
+  module QueueAdapters
+    class CloudtaskerAdapter #:nodoc:
+      class Worker #:nodoc:
```

Comment:
Also please add doc on the purpose of JobWrapper.

Reply:
Got it.